### PR TITLE
ADDED: Institutions page added with responsiveness.

### PR DESCRIPTION
**Pull request by XRegiGigaSX**
This commit contains Institutions page with the responsiveness.

### CODE REVIEW

File: src/Components/Institutions/institutions.scss [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  position: relative;
+  height: 100%;
+  width: 34%;
+}
+
+.title-search-box-search {
+  height: 3.4em;
+  width: 100%;
+  padding: 0.5em;
+  border-radius: 5px;
```
Comment (resyfer) [inline src/Components/Institutions/institutions.scss]: Border radius should not use absolute units (mostly nothing should use absolute units) as curvature would then depend on resolution. Fix it, and everywhere similar stuff is used

File: src/Components/Institutions/institutions.scss [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  }
+  .insti-card {
+    width: 33vw;
+    height: 33vw;
+    padding: 2%;
+    margin: 4% 1.6%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-shadow: 0px 0px 15px 0px rgb(138, 138, 138);
```
Comment (resyfer) [inline src/Components/Institutions/institutions.scss]: Absolute unit issue again here

**Review by resyfer — CHANGES_REQUESTED** [review]

File: src/Components/Institutions/institutions.scss (hunk 1 above)
Comment (XRegiGigaSX) [reply src/Components/Institutions/institutions.scss]: Made necessary changes.

File: src/Components/Institutions/institutions.scss (hunk 2 above)
Comment (XRegiGigaSX) [reply src/Components/Institutions/institutions.scss]: Made necessary changes

**Review by resyfer — APPROVED** [review]

**Pull request merged**